image/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openjdk:17-jdk-slim
+
+RUN apt update && \
+    apt install -y unzip wget && \
+    mkdir /grasscutter && cd /grasscutter && \
+    wget -O grasscutter.jar https://github.com/Grasscutters/Grasscutter/releases/download/v1.4.3/grasscutter-1.4.3.jar && \
```

Comment:
Instead of specifically specifying `1.4.3`, add a variable for it? (not completely familiar with `Dockerfile`)